recipe/meta.yaml

```diff
@@ -1,110 +1,104 @@
-  # https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/21220961#L972
-  skip: true  # [win and py27]
-  script:
-    # Most of this script is adapted from the pytorch/pythorch-cpu
-    # meta.yaml.template build.sh and bld.bat scripts
-    - {{ export }} TN_BINARY_BUILD=1
-    - {{ export }} PYTORCH_BINARY_BUILD=1
-    - {{ export }} NO_CUDA=1
-    - {{ export }} PYTORCH_BUILD_VERSION={{ PKG_VERSION }}
-    - {{ export }} PYTORCH_BUILD_NUMBER={{ PKG_BUILDNUM }}
-    - {{ export }} BUILD_CUSTOM_PROTOBUF=ON
-    # Use ninja as the build just won't finish on windows
-    - {{ export }} CMAKE_GENERATOR=Ninja
-    # I have no idea what this flag does, but they recommend turning it on
-    # when we don't find three of their headers.
-    # It seemed to be required on Appveyor, but not azure, strange???
-    # - {{ export }} GEN_TO_SOURCE=1                          # [win]
-    # Why are all warnings treated as errors???
-    - export CXXFLAGS="-Wno-error=unused-result $CXXFLAGS"  # [linux]
-    - export CFLAGS="-Wno-error=unused-result $CFLAGS"      # [linux]
-    # Disable this other 3rd party binary
-    - {{ export }} NO_MKLDNN=1
-    # - {{ export }} NO_TEST=1
-    # I couldn't find any documentation on this, but eventually they call
-    # a script tools/build_pytorch_libs.sh
-    # whcih passes EXTRA_CAFFE2_CMAKE_FLAGS at the end of the cmake command
-    # to all 3rd party libraries.
-    - {{ export }} EXTRA_CAFFE2_CMAKE_FLAGS="-DUSE_MPI=OFF -DUSE_NUMA=OFF -DUSE_NCCL=OFF -DATEN_NO_TEST=OFF"
-    # Why do I need to export these?
-    #
-    # The CMake for onnx reports:
-    # therefore, it clearly has a bug somewhere
-    # --   Protobuf compiler     : /home/mark2/miniconda3/envs/compile/bin/protoc
-    # --   Protobuf includes     : /usr/include
-    # --   Protobuf libraries    : /home/mark2/miniconda3/envs/compile/lib/libprotobuf.so;-lpthread
-    - export CXXFLAGS="-I{{ PREFIX }}/include/ $CXXFLAGS"  # [unix]
-    - export CFLAGS="-I{{ PREFIX }}/include/ $CFLAGS"  # [unix]
-    # How do we add /std:c++11 to windows CFLAGS
-    # There is an other weird python package called ninja
-    # which is their preferred method of building
-    # that said, it does nothing but call "ninja"
-    # - pip install ninja
-    - "{{ PYTHON }} -m pip install . --no-deps -vv --disable-pip-version-check"
+  string: cuda{{ cudatoolkit | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [pytorch_variant == "gpu"]
+  string: cpu_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                      # [pytorch_variant == "cpu"]
+  detect_binary_files_with_prefix: False
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - cmake
-    - ninja
-    - make  # [unix]
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - git       # [not win]
+    - m2-git    # [win]
   host:
+    - cmake
+    - git       # [not win]
+    - m2-git    # [win]
+    # GPU requirements
```

Comment:
Ah got it, it is still a WIP and you are leaving the GPU stuff commented out while the CPU mgiration is completed.